src/main/scala/dotty/xml/interpolator/package.scala

```diff
@@ -5,5 +5,5 @@ import scala.quoted._
 type Scope = scala.xml.NamespaceBinding
 implicit val top: Scope = scala.xml.TopScope
 
-inline def (ctx: => StringContext) xml (args: => (given Scope => Any)*) given Scope <: Any =
```

Comment:
`Scope <: Any` is no longer permitted – @nicolasstucki can we just drop it?

Reply:
The `<: Any` part is the result type of the inline method, so it should now be written `inline def ...(given Scope) <: Any = `

Reply:
Oh. Well... Good thing we now need parentheses around givens to avoid such a confusion.

Reply:
Clearly the old syntax was confusing